cvat-ui/src/containers/register-page/register-page.tsx

```diff
@@ -1,38 +1,80 @@
-import React from 'react';
-import { connect } from 'react-redux';
-import { registerAsync } from 'actions/auth-actions';
-import RegisterPageComponent from 'components/register-page/register-page';
+import React, { ComponentType } from 'react';
+import { connect, ResolveThunks } from 'react-redux';
 import { CombinedState } from 'reducers/interfaces';
+import { registerAsync } from '../../modules/auth/authActions';
+import { RouteComponentProps, withRouter } from 'react-router';
+import { Col, Row } from 'antd';
+import Title from 'antd/lib/typography/Title';
+import RegisterForm, { RegisterData } from './register-form';
+import Text from 'antd/lib/typography/Text';
+import { Link } from 'react-router-dom';
+import { compose } from 'redux';
 
 interface StateToProps {
     fetching: boolean;
 }
 
 interface DispatchToProps {
-    onRegister: (username: string, firstName: string,
-        lastName: string, email: string,
-        password1: string, password2: string) => void;
+    onRegister: typeof registerAsync;
 }
 
-function mapStateToProps(state: CombinedState): StateToProps {
-    return {
-        fetching: state.auth.fetching,
+type Props = StateToProps & ResolveThunks<DispatchToProps> & RouteComponentProps;
+
+function RegisterPageContainer(props: Props) {
+    const sizes = {
+        xs: { span: 14 },
+        sm: { span: 14 },
+        md: { span: 10 },
+        lg: { span: 4 },
+        xl: { span: 4 },
     };
+
+    const {
+        fetching,
+        onRegister,
+    } = props;
+
+    return (
+        <Row type='flex' justify='center' align='middle'>
+            <Col {...sizes}>
+                <Title level={2}> Create an account </Title>
+                <RegisterForm
+                    fetching={fetching}
+                    onSubmit={(registerData: RegisterData): void => {
+                        onRegister(
```

Comment:
I suppose using object here will be better

Reply:
It was just moved code without any changes